src/assets/scss/_markdown-container.scss

```diff
@@ -101,6 +101,18 @@
     margin: 2px;
   }
 
+  // in atlas-d2k, some iframe content has a <h3> then <p> tag before the iframe
```

Comment:
I think the names that we're choosing all should have the `chaise-` namespace. Also let's not talk about specific deployments in our code.

We should also list these classes and then find appropriate names for all of them.

Reply:
This relates to the other comment I made about discussing where we want these classes to be defined.

While defining these in two places gives us flexibility, it might confuse data modelers. It would be harder for them to determine if a class is available throughout chaise or just markdown.